openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -16,6 +16,121 @@
         }
         return newArr;
     }
+
+    function adaptiveFilterPoints(charts) {
+
+        var convertToAdaptiveBytes = function(charts, type, i, multiplier, unit) {
+            charts[type].y[i] = Math.round((charts[type].y[i] * multiplier)*100)/100;
+            charts[type].hovertemplate[i] = charts[type].y[i] + ' ' + unit;
+        };
+
+        for(var i=0; i<charts[0].y.length; i++)
+        {
```

Comment:
```suggestion
        for(var i=0; i<charts[0].y.length; i++) {
```

Why suddenly the style of indenting these curly braces is changed? Can you maintain consistency with the rest of the code please?

Applies for all the other lines as well.